test/workbox-streams/integration/test.js

```diff
@@ -0,0 +1,41 @@
+const {expect} = require('chai');
```

Comment:
Creating meaningful `node`-based mocks for all the primitives needed by `workbox.streams` wasn't feasible, so I'm relying on the integration tests to exercise most of the actual logic.